lightning/src/blinded_path/payment.rs

```diff
@@ -61,6 +61,9 @@ pub struct ForwardTlvs {
 	///
 	/// [`BlindedHop::encrypted_payload`]: crate::blinded_path::BlindedHop::encrypted_payload
 	pub features: BlindedHopFeatures,
+	/// Set if this [`BlindedPath`] is concatenated to another, to indicate the
+	/// [`BlindedPath::blinding_point`] of the appended blinded path.
+	pub next_blinding_override: Option<PublicKey>,
```

Comment:
I think Rusty pointed out here that it can help prevent the intro node from being aware that it's the intro node: https://github.com/lightning/bolts/pull/1182#discussion_r1695943553? I think this is clarified in https://github.com/lightning/bolts/pull/1182 but still need to review that.

Reply:
Yes, using `next_blinding_override` in OMs has always been required to prepend the unblinded path to the intro node.

Reply:
Oops, I think I misread your message. I'm not sure I follow Rusty's logic, actually. It seems like creating a blinded path from us to the intro node trades off the intro node knowing that they're the intro node for the node prior to the intro node knowing that the next hop is the intro node(?). Unless we set `next_blinding_override` for every hop in our prepended blinded path, which I suppose is possible.